pkg/cmd/server/api/helpers.go

```diff
@@ -305,6 +305,7 @@ func GetKubeClient(kubeConfigFile string) (*kclient.Client, *restclient.Config,
 	// TODO: this should be configured by the caller, not in this method.
 	kubeConfig.QPS = 100.0
 	kubeConfig.Burst = 200
+	kubeConfig.ContentType = "application/vnd.kubernetes.protobuf"
```

Comment:
So on the reading side, for 1.3 we'll get?
 1. empty = json
 2. specified is respected.

On the reading side for 1.4, we'll get?
 1. empty = protobuf
 2. specified is respected

We only get to do one change like this.  After that we'll have some files written with a specified ContentType.